GameDataParser/Program.cs

```diff
@@ -65,7 +65,9 @@ private static async Task Main()
                 new FurnishingShopParser(resources),
                 new HomeTemplateParser(resources),
                 new MasteryUGCHousingParser(resources),
-                new JobParser(resources)
+                new JobParser(resources),
+                new RewardContentParser(resources),
+                new OXQuizParser()
```

Comment:
Does it not require resource?

Reply:
No, it's not from the game data. This is all server-sided data that I grabbed from a ms2 wiki